releasenotes/notes/0.5/experiment_data_fixes-f69c3569a8ba1342.yaml

```diff
@@ -0,0 +1,17 @@
+---
+upgrade:
```

Comment:
The upgrade section is for changing behavior that users need to know when upgrading, not for improvements. These should go into new features or other notes.